test/e2e/volume_provisioning.go

```diff
@@ -184,6 +186,70 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			Expect(err).To(HaveOccurred())
 			framework.Logf(err.Error())
 		})
+
+		It("should test that deleting a claim before the volume is provisioned deletes the volume. [Volume]", func() {
```

Comment:
Barring regression, this test completes in under 5 minutes.  So according to the description for the tag, I think it can be omitted.